.idea/misc.xml

```diff
@@ -1,15 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="EntryPointsManager">
-    <list size="5">
-      <item index="0" class="java.lang.String" itemvalue="org.terasology.entitySystem.event.ReceiveEvent" />
-      <item index="1" class="java.lang.String" itemvalue="org.terasology.entitySystem.systems.RegisterSystem" />
```

Comment:
Why is `RegisterSystem` not needed (anymore)? Is the class considered used in any case?

Reply:
I was re-checking these at some point and removing the class-level annotation didn't result in any inspection warning popping up. 🤷